test/server/Server.test.js

```diff
@@ -115,50 +104,4 @@ describe("Server", () => {
       utilSpy.mockRestore();
     });
   });
-
-  // issue: https://github.com/webpack/webpack-dev-server/issues/1724
-  describe("express.static.mime.types", () => {
```

Comment:
This was added for a workaround logic that doesn't exist anymore. https://github.com/webpack/webpack-dev-server/pull/1580/